test/CMakeLists.txt

```diff
@@ -9,6 +10,13 @@ set_property( GLOBAL PROPERTY USE_FOLDERS ON )
 set( CMAKE_C_STANDARD 90 )
 set( CMAKE_C_STANDARD_REQUIRED ON )
 
+# If no configuration is defined, turn everything on.
```

Comment:
Nit : The comment would be better with "If either configuration is not defined,  turn everything on"